<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatWindow</class>
 <widget class="QDialog" name="GroupChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="messageListWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>398</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageInput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUserButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
